timestamp,file_status,status
31/07/2023 08:16:07.768541,source/CALL_PROCEDURE_get_employee_details_to_display_emp_details.sql,"ERROR:  syntax error at or near ""integer""
LINE 2:     v_employee_id integer := 1;
                          ^
"
31/07/2023 08:16:09.664977,source/CREATE_OR_REPLACE_PROCEDURE_get_employee_details_to_get_emp_details.sql,"ERROR:  syntax error at or near ""BEGIN""
LINE 8: BEGIN
        ^
"
31/07/2023 08:16:11.831688,source/CREATE_OR_REPLACE_TRIGGER_tr_Employee_Details_to_show_changes.sql,"ERROR:  syntax error at or near ""ENABLE""
LINE 21: ENABLE
         ^
"
31/07/2023 08:16:14.944421,source/CREATE_OR_REPLACE_trigger_tr_Employee_Details_Changes_to_store_changes.sql,"ERROR:  syntax error at or near "":""
LINE 15:     VALUES(NEW.emp_name, :OLD.emp_name, v_date,'Update');
                                  ^
"
31/07/2023 08:16:17.092382,source/CREATE_SEQUENCE_demo_sequence1.sql,"ERROR:  syntax error at or near ""ORDER""
LINE 8: ORDER;
        ^
"
31/07/2023 08:16:19.338982,source/CREATE_SEQUENCE_demo_sequence2.sql,"ERROR:  syntax error at or near ""CACHE""
LINE 7: NO CACHE
           ^
"
31/07/2023 08:16:20.821176,source/CREATE_TABLE_Employee_Details.sql,"ERROR:  relation ""Employee_Details"" already exists
"
31/07/2023 08:16:22.388372,source/CREATE_TABLE_Employee_Details_Changes_to_store_changes.sql,"ERROR:  relation ""Employee_Details_Changes"" already exists
"
31/07/2023 08:16:23.713567,source/DELETE_FROM_Employee_Details.sql,Success
31/07/2023 08:16:26.567559,source/INSERT_INTO_Employee_Details_new_data.sql,Success
31/07/2023 08:16:29.883916,source/INSERT_initial_data_INTO_Employee_Details.sql,Success
31/07/2023 08:16:31.308521,source/UPDATE_Employee_Details.sql,Success
